javascript/javascript1/week4/ticTacToe.js

```diff
@@ -0,0 +1,54 @@
+// const position = [
+//   ['o', ' ', ' '],
+//   ['o', ' ', ' '],
+//   ['o', ' ', ' ']
+// ];
+// const position = [
+//   [' ', ' ', 'o'],
+//   [' ', ' ', 'o'],
+//   [' ', ' ', 'o']
+// ];
+// const position = [
+//   ['o', 'o', 'o'],
+//   [' ', ' ', ' '],
+//   [' ', ' ', ' ']
+// ];
+// const position = [
+//   [' ', ' ', ' '],
+//   ['o', 'o', 'o'],
+//   [' ', ' ', ' ']
+// ];
+// const position = [
+//   [' ', ' ', ' '],
+//   [' ', ' ', ' '],
+//   ['o', 'o', 'o']
+// ];
+// const position = [
+//   ['o', ' ', ' '],
+//   [' ', 'o', ' '],
+//   [' ', ' ', 'o']
+// ];
+// const position = [
+//   [' ', ' ', 'o'],
+//   [' ', 'o', ' '],
+//   ['o', ' ', ' ']
+// ];
+const position = [
+    [' ', 'x', ' '],
+    [' ', 'x', ' '],
+    [' ', 'x', ' ']
+  ];
+  // console.log(position[0][1])
+  // console.log(position[1][1])
+  // console.log(position[2][1])
+  function hasWon(position, player){
+      if( position[0][1] === player && position[1][1] === player  && position[2][1] === player ){
+          return true
+      }else if( position[0][2] === player && position[1][1] === player  && position[2][0] === player ){
+          return true
+      }
+      // add more else if conditions here
+      return false
+  }
+  console.log(hasWon(position, 'x'))
```

Comment:
again, these console.logs are very dry. You will just output true or false. Console.logs are for YOU, so make them readable.